beacon-chain/blockchain/service.go

```diff
@@ -226,6 +196,22 @@ func (c *ChainService) updateHead(slot uint64) {
 	c.candidateCrystallizedState = nilCrystallizedState
 }
 
+// doesPoWBlockExist checks if the referenced PoW block exists.
+// If the node is connected to a PoW chain, defaults to true.
```

Comment:
doesn't this code return true if it is _not_ connected to a PoW chain?

Reply:
Ah, yep. Good eye!